pkg/cmd/server/origin/openshift_apiserver.go

```diff
@@ -95,6 +95,7 @@ type OpenshiftAPIExtraConfig struct {
 	RegistryHostnameRetriever          imageapi.RegistryHostnameRetriever
 	AllowedRegistriesForImport         *configapi.AllowedRegistries
 	MaxImagesBulkImportedPerRepository int
+	AdditionalTrustedCA                string
```

Comment:
At this point we should have bytes

Reply:
what's the motivation for that?

Reply:
> what's the motivation for that?

containability and testability.  We have a serializable config focus on human users, but the next step is to transform that into a runnable, nonserializable struct.  This is that struct.  Passing serializeable data that needs more transformation is against that goal.

Reply:
addressed in `david2`